### PR TITLE
fix: add isLoading prop to Button component

### PR DESCRIPTION
**Pull request by Akirtovskis**

### CODE REVIEW

File: client/src/ui/components/Button.tsx [hunk 1]
```diff
@@ -44,7 +47,15 @@ export const Button = ({
     )}
   >
     {iconStart && <span className="mr-3">{iconStart}</span>}
-    {children}
+    {isLoading ? (
+      <React.Fragment>
+        <Spinner size="extraSmall" />
+        <span className="ml-2 opacity-50"> {children}</span>
```
Comment (pradel) [inline client/src/ui/components/Button.tsx:53]: there is a space in the `span` I think it's better to remove it

File: client/src/ui/components/Button.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import cx from 'classnames';
+import { Spinner } from './Spinner';
 
 interface ButtonProps extends React.HTMLProps<HTMLButtonElement> {
   iconStart?: React.ReactNode;
   iconEnd?: React.ReactNode;
   color: 'red' | 'grey';
   variant?: 'solid' | 'outline';
+  isLoading?: boolean;
```
Comment (pradel) [inline client/src/ui/components/Button.tsx:10]: Wondering if it makes sense to set `disabled` to true automatically if `isLoading` is true

File: client/tailwind.config.js [hunk 3]
```diff
@@ -8,6 +8,11 @@ module.exports = {
         sans: ['Inter var', ...defaultTheme.fontFamily.sans],
       },
     },
+    minWidth: {
```
Comment (pradel) [inline client/tailwind.config.js:11]: this can be reverted as we don't use it anymore :)

File: client/src/ui/components/Button.tsx (hunk 2 above)
Comment (Akirtovskis) [reply client/src/ui/components/Button.tsx:10]: @pradel this is indeed a very good idea. Updated and refactored where it was necessary.

**Review by pradel — APPROVED** [review]

**Pull request merged**